examples/packet.arm64.yml

```diff
@@ -1,3 +1,9 @@
+kernel:
+  image: linuxkit/kernel:4.9.76
```

Comment:
I guess it is not possible to set `kernel.foo` without also setting `kernel.image`? This could be fixed up in the moby tool which already does conditional assignment of subfields for various things (runtime is one I remember adding myself).